devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy

```diff
@@ -113,6 +112,8 @@ def poll() {
  * PING is used by Device-Watch in attempt to reach the Device
  * */
 def ping() {
+	unschedule()
```

Comment:
`configureHealthCheck` will call `unschedule` so we don't want to also call it here. If we do and the health check was already configured then it would be left unscheduled.